parent/pom.xml

```diff
@@ -734,12 +734,6 @@
         <version>${project.version}</version>
       </dependency>
 
-      <dependency>
-        <groupId>io.camunda</groupId>
-        <artifactId>tasklist-importer-870</artifactId>
```

Comment:
these references have been removed by tasklist from main already... we will need to rebase